cdap-explore/src/test/java/co/cask/cdap/explore/service/ExploreServiceTestsSuite.java

```diff
@@ -31,12 +33,17 @@
   HiveExploreServiceTestRun.class,
   WritableDatasetTestRun.class,
   HiveExploreObjectMappedTableTestRun.class,
-  HiveExploreServiceStopTestRun.class
```

Comment:
Since `HiveExploreServiceStopTestRun` stops the service, it is a good idea not to run it as part of the test suite.